src/linker/Linker/Driver.cs

```diff
@@ -715,6 +723,29 @@ public bool Run (ILogger customLogger = null)
 
 		partial void PreProcessPipeline (Pipeline pipeline);
 
+		private static HashSet<uint> ParseWarnings (string value)
+		{
+			string Unquote (string arg)
+			{
+				if (arg.Length > 1 && arg[0] == '"' && arg[arg.Length - 1] == '"')
+					return arg.Substring (1, arg.Length - 2);
+
+				return arg;
+			}
+
+			value = Unquote (value);
+			string[] values = value.Split (new char[] { ',', ';', ' ' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Can you check what CSC supports as separators? Would be good to be consistent.

Reply:
This is the same set that they use for separators: https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs#L1956